Doc/whatsnew/3.10.rst

```diff
@@ -596,6 +602,24 @@ Changes in the Python API
   a 16-bit unsigned integer.
   (Contributed by Erlend E. Aasland in :issue:`42393`.)
 
+* The ``loop`` parameter to nearly all :mod:`asyncio` functions has been removed
+  following deprecation in Python 3.8. For example, a coroutine that currently
+  look like this::
+
+     import asyncio
+
+     async def foo(loop):
+         await asyncio.sleep(loop=loop)
+
+  Can *usually* be replaced with this::
```

Comment:
Are there actually cases where this example is not the solution and another example should be provided? The APIs that I'm familiar with that got changed would be solved by this in every case, but maybe there are some that require a different fix?

If not, I'd remove the `*usually*` so there's no confusion about what else might need to be done.

Reply:
It specifically doesn't apply if the user wants to utilize an event loop other than the currently running one. But, for 90% of use cases, the user will want to. I expanded upon this in [another comment](https://github.com/python/cpython/pull/24256#discussion_r560511567).